developers/academy/zero_to_mvp/103_schema_and_imports/40_example.mdx

```diff
@@ -140,7 +140,7 @@ We can now add the class to the schema.
 <FilteredTextBlock
   text={PythonCodeExample}
   startMarker="# Define the class"
-  endMarker='# End class definition'
+  endMarker='# Finished creating the class'
```

Comment:
Hey, this is to pull up a code snippet based on these markers. So we can change these, but we'll have to change the corresponding comment in the code. :) 

I propose to leave this one as is for now.

Reply:
Oh my bad ;) I see that also comes from the code :)